doc/rho.1

```diff
@@ -217,11 +293,92 @@ Any or all profiles can be deleted using the 'clear' subcommand.
 .B rho profile clear --name=
 .I name
 .B | --all
-
+.B [--vault=
+.I vault_file
+.B ]
+.PP
+.TP
+--name=name
+This argument gives the profile to delete.
+.PP
+.TP
+--all
+This deletes all stored profiles.
+.PP
+.TP
+--vault=vault_file
+This contains the path and filename of the file containing the vault password. If this option is used the file should be limited in access on the system.
+.PP
+.PP
+.SS FACTS
+The 'fact' command is used to understand information that can be reported or to alter the contents of a report created from the 'rho scan' command.
+.PP
+.B
+Listing Facts
+.PP
+A list of facts that can be gathered during the scanning process can be obtained with the 'list' command.
+.PP
+.B rho fact list
+.B [--filter=
+.I reg_ex
+.B ]
+.TP
+--filter=reg_ex
+Optionally, provide a filter view of the list of facts with a regular expression -- e.g 'uname.*'.
+.PP
+.B
+Redacting Facts
+.PP
+Sensitive facts can be removed from a report CSV file using the 'redact' command. The facts that are removed with this command are:
+.I connection.host, connection.port, uname.all,
+and
+.I uname.hostname.
+.PP
+.B rho fact redact
+.B --reportfile=
+.I file
+.PP
+.TP
+--reportfile=file
+The path and filename of the comma-separated values (CSV) file to alter.
+.PP
+.B
+Encrypting Facts
+.PP
+Sensitive facts can be encrypted within a report CSV file using the 'encrypt' command. The facts that are encrypted with this command are:
```

Comment:
For this and `rho decrypt`, I think people are going to wonder what key is used for encryption/decryption.

Reply:
+1

Also I think could be mentioned about using a password to encrypt and decrypt the information

Reply:
I added some information around the password usage in encrypt and decrypt.